low-level/solution/lr2/server/server_resource_map.py

```diff
@@ -82,3 +82,15 @@ def get_node_info(node):
         inst = res.groups()[1] if res.groups()[1] else "*"
         node = res.groups()[0] if res.groups()[1] else res.groups()[2]
         return node, inst
+
+    @staticmethod
+    def _sort_list_of_dict_by_kv(data, sort_key, key_remove=False):
```

Comment:
name for _sort_list_of_dict_by_kv() can be better, like sort_by_specific_kv()

Reply:
Done